modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyBranch.jsx

```diff
@@ -200,8 +226,21 @@ function VocabularyBranch(props) {
       }
 
       {/* Term name */}
-      <Typography onClick={() => loadTerm(id, path)}
+      <Typography onClick={(evt) => {(evt.target.type != "checkbox") && loadTerm(id, path)}}
                   className={classes.infoName + (focused ? (" " + classes.focusedTermName) : " ")}>
+        {/* Browser term select tools */}
+	    { addCheckbox &&
+	      <Checkbox
+	        color="primary"
```

Comment:
IMO the color of the browser checkboxes and radios should be consistent with the form controls. In the form we have orenge-red (secondary color) while in the browser we have midnight blue (primary color). I suggest changing it to secondary, wdyt? @sashaandjic @veronikaslc 
```suggestion
	        color="secondary"
```